Translations/translation_ZH_TW.json

```diff
@@ -88,19 +88,19 @@
 	"menuOptions": {
 		"DCInCutoff": {
 			"text2": "電源",
-			"desc": "輸入電源；設定自動停機電壓 <DC 10V> <S 鋰電池，以每顆3.3V計算；此設定會停用功率限制>"
+			"desc": "輸入電源；設定自動停機電壓 (DC 10V) (S 鋰電池，以每顆3.3V計算 | 此設定會停用功率限制)"

```

Comment:
@alvinhochun
Sorry Sir, but are you referring to this change here?
Or / and ...